R/make_simspin_file.R

```diff
@@ -74,7 +76,8 @@ make_simspin_file = function(filename, cores=1, disk_age=5, bulge_age=10,
 
   if (write_to_file){
     if (missing(output)){
-    output = paste(sub('\\..*', '', filename), "_", temp_name, ".Rdata", sep="")
+    output = paste(sub('\\..*', '', filename), "_", temp_name, ".Rdata", sep="") # CHANGE THIS!!!
```

Comment:
```suggestion
    output = paste(sub('\\..*', '', filename), "_", temp_name, ".Rdata", sep="") 
```
Probably can remove the comment from the above line and the commented out alternative?